### PR TITLE
implement event consumer similarly to the dp-kafka consumer example

### PR DESCRIPTION
**Pull request by DavidSubiros**
### What

- Removed message consumer and interface and Consumer struct, as they duplicate dp-kafka functionality
  - MessageConsumer replaced with kafka.IConsumerGroup interface
- Added KafkaNumWorkers config value (default 1) to allow more than 1 consumer if required.
- Consume loop(s) end when the kafka consumer Upstream or closer channels are closed.
- Removed close function, as the consumer loop will exit when kafka consumer 'StopListen' is called
- Added Kafka consumer logger go-routine in service

### How to review
- Make sure code changes make sense
- Make sure unit tests pass
- (optional) you can test the kafka production and consumption by running at the same time:
  - dp-compose/run.sh
  - dp-hello-world-event : make debug
  - dp-hello-world-event/cmd/producer : go run main.go
  - then type something in the producer command line and check that it is correctly consumed.

### Who can review

anyone

### CODE REVIEW

**Review by rav-pradhan — COMMENTED** [review]
Tested locally and looks fine. In terms of code I think it all makes sense but maybe worth getting a second review from someone more familiar with Kafka?

File: dp-hello-world-event/config/config.go
Comment (CarlHembrough) [inline dp-hello-world-event/config/config.go:18]: The details of this should be added to the readme

**Review by CarlHembrough — APPROVED** [review]
LGTM 👍

**Review by CarlHembrough — APPROVED** [review]

**Pull request merged**
